### PR TITLE
cmd: update misc. command references

### PR DESCRIPTION
**Pull request by ianvernon**
* cmd: update `cilium bpf ipcache list` Short description
* cmd: update monitor command Short description

Signed-off by: Ian Vernon <ian@cilium.io>

Related-to: #3597

### CODE REVIEW

File: cilium/cmd/monitor.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ const (
 // monitorCmd represents the monitor command
 var monitorCmd = &cobra.Command{
 	Use:   "monitor",
-	Short: "Monitoring",
```
Comment (raybejjani) [inline cilium/cmd/monitor.go:47]: The consumable is consumed by consumers... the monitor provides monitoring for monitors.
More seriously, it's probably obvious but "display endpoint BPF program network events" might help illustrate where in the stack this is getting it's data (and avoid using "monitor" again). This is a bike-shedding nit, however.

**Review by raybejjani — APPROVED** [review]

File: cilium/cmd/monitor.go (hunk 1 above)
Comment (ianvernon) [reply cilium/cmd/monitor.go:47]: Monitoring is fairly self-evident though - you are monitoring BPF program events as they occur, vs. the abstract nature of a `Consumable` ;)

Comment (ianvernon) [reply cilium/cmd/monitor.go:47]: I agree though, I don't like self-referential definitions. Will fix.

**Comment by ianvernon** [comment]
test-me-please

File: cilium/cmd/monitor.go (hunk 1 above)
Comment (ianvernon) [reply cilium/cmd/monitor.go:47]: Fixed in latest upload :)

Comment (raybejjani) [reply cilium/cmd/monitor.go:47]: hehe, yeah, I didn't think what I said was particularly better. +1 to whatever you decide.

File: Documentation/cmdref/cilium.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ CLI for interacting with the local Cilium Agent
 * [cilium endpoint](cilium_endpoint.html)	 - Manage endpoints
 * [cilium identity](cilium_identity.html)	 - Manage security identities
 * [cilium kvstore](cilium_kvstore.html)	 - Direct access to the kvstore
-* [cilium monitor](cilium_monitor.html)	 - Monitoring
+* [cilium monitor](cilium_monitor.html)	 - display BPF program events
```
Comment (aanm) [inline Documentation/cmdref/cilium.md]: `s/display/Display`

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-docs-please

File: Documentation/cmdref/cilium.md (hunk 2 above)
Comment (ianvernon) [reply Documentation/cmdref/cilium.md]: fixed.

**Review by joestringer — APPROVED** [review]

**Review by aanm — APPROVED** [review]

**Pull request merged**